suites/nautilus/ansible/sanity_containerized_ceph_ansible.yaml

```diff
@@ -55,38 +55,38 @@ tests:
             branch: luminous
             role: mon
       desc: Test librbd unit tests
-   - test:
-      name: rbd cli image
-      module: rbd_system.py
-      config:
-            test_name: cli/rbd_cli_image.py
-            branch: master
-      polarion-id: CEPH-83572722
-      desc: CLI validation for image related commands
-   - test:
-      name: rbd cli snap_clone
-      module: rbd_system.py
-      config:
-            test_name: cli/rbd_cli_snap_clone.py
-            branch: master
-      polarion-id: CEPH-83572725
-      desc: CLI validation for snap and clone related commands
-   - test:
-      name: rbd cli misc
-      module: rbd_system.py
-      config:
-            test_name: cli/rbd_cli_misc.py
-            branch: master
-      polarion-id: CEPH-83572724
-      desc: CLI validation for miscellaneous rbd commands
-   - test:
-      name: rbd cli import_export
-      module: rbd_system.py
-      config:
-            test_name: cli/rbd_cli_import_export_diff.py
-            branch: master
-      polarion-id: CEPH-83572723
-      desc: CLI validation for import, export and diff related commands
+  #  - test:
+  #     name: rbd cli image
```

Comment:
same, lets remove the unneeded comments

Reply:
Lets take this in next PR.

Reply:
okay , once #86 is fixed will enable them